src/Observable.ts

```diff
@@ -29,11 +28,9 @@ export class Observable<EventSpec> {
     eventName: K,
     handler: TEventCallback<E>
   ): VoidFunction;
+  on(handlers: EventRegistryObject<EventSpec>): VoidFunction;
   on<K extends keyof EventSpec, E extends EventSpec[K]>(
-    handlers: EventRegistryObject<K, E>
```

Comment:
Why did you remove this overload?
The base signature is not exposed by TS to the consumer, is it?

Reply:
It looks removed but it isn't, look at the final code on the right
![Screenshot 2023-07-29 at 13 00 54](https://github.com/fabricjs/fabric.js/assets/10067273/efa530a8-3ced-49f5-bcf2-016483e0b875)